xbmc/filesystem/AudioBookFileDirectory.cpp

```diff
@@ -137,6 +299,25 @@ bool CAudioBookFileDirectory::GetDirectory(const CURL& url,
   return true;
 }
 
+void CAudioBookFileDirectory::AddCommaDelimitedString(const std::vector<std::string>& data,
```

Comment:
Naming? This seems specific to roles.

Reply:
Kind of.  But it adds a comma delimited string to the tag.  So.......